R/main_script.R

```diff
@@ -1493,7 +1493,8 @@ get_plen_comm_details <- function(date_range_from,date_range_to,fact,plen_comm="
                                      id_mp = list("contact", 1, "id")) %>%
                         dplyr::distinct() %>%
                         dplyr::mutate(id_verg=as.character(id_verg),
-                                      id_fact = as.integer(id_fact)), by=c("id_fact"="id_fact","id_verg"="id_verg")) -> result
+                                      id_fact = as.integer(id_fact)), by=c("id_fact"="id_fact","id_verg"="id_verg")) %>%
+                        dplyr::mutate(omschrijving = gsub("<(.|\n)*?>", "", omschrijving)) -> result
```

Comment:
Hier nog iets abnormaal gemerkt, bv bij een call op data van 2000 geeft geen resultaten terug (maar in de UI, is dit wel te vinden bv: https://www.vlaamsparlement.be/nl/parlementaire-documenten/parlementaire-initiatieven/273422)